src/regular-expression-comment.ts

```diff
@@ -25,35 +25,41 @@ export default {
           const regularExpressionLine = node.loc.start.line;
           const previousLine = regularExpressionLine - 1;
           const previousLineComment = lines[previousLine - 1];
-          const regularExpressionComment = comments.find((comment) => {
-            if (!comment.loc) {
-              return false;
-            }
-            // This regex is to check if the comment has valid text
-            const regularExpression = /[a-zA-Z]/gu;
-            // This regex is to check if the line starts with or without spaces and followed by two or more consecutive slashes // or start with /* and may have one or more asterisks, continuing until the first occurrence of */.
-            const commentRegularExpressionLine = /^\s*(?:\/{2,}|\/\*+)/gu;
-            const hasComment = regularExpression.test(comment.value.trim());
-            if (comment.type === 'Line' || comment.loc.start.line === comment.loc.end.line) {
+
+          // Check if the regular expression contains any simple text
+          const isSimpleTextPattern = /^[0-9:./,\sa-zA-Z-]*$/gu.test(node.value.source);
+
+          if (!isSimpleTextPattern) {
```

Comment:
i think it would be better to do a `if (isSimpleTextPattern)` and just return.  makes the diff easier to follow too.